packages/googlecloud/functions/getanswer/main.py

```diff
@@ -115,8 +121,15 @@ def getanswer(request):
 
     end = time.time()
     elapsed = int((end - start) * 1000)
+
+    # 
+    return(answer)
+
+    # disabled for local deployment, reenable for web 
```

Comment:
commenting out gcloud functionality for local getanswer deployment, and changing what comes back so it's not posting to a db - just saving it here for reference, but should be undone before full deployment